### ISSUE TITLE
Error "The application requires that assembly netfx.force.conflicts" after upgrading to v5.0.0

### ISSUE THREAD

**Issue by martani** [JoeMayo/LinqToTwitter#145]
I upgraded to linqtotwitter v5.0.0 and I'm getting the following error:

> Unable to install or run the application. The application requires that assembly netfx.force.conflicts Version 0.0.0.0 be installed in the Global Assembly Cache (GAC) first.

These are the references I have for linqtotwitter in my csproj:
```
<Reference Include="LinqToTwitter.netstandard, Version=5.0.0.0, Culture=neutral, processorArchitecture=MSIL">
  <HintPath>..\packages\linqtotwitter.5.0.0\lib\net461\LinqToTwitter.netstandard.dll</HintPath>
</Reference>
```

Other [issues](https://github.com/dotnet/standard/issues/704) like this online seem to be related to `netstandard`. Is there probably some dependencies I'm missing?

Thanks.

**Comment by martani** [comment]
Yes I'm using WPF.

**Comment by JoeMayo** [comment]
Haven't reproduced this yet. I set up a new WPF project and added nuget ref for v5.0.0, but it's working for me. Not sure what's happening.

**Comment by martani** [comment]
Sorry I forgot to mention. The error happen after you publish the app as a click once app (right click on the project and choose "publish" on visual studio in retail mode). If you run it from the published location (running the ".application" file) you get this error.

**Comment by martani** [comment]
I was able to bypass this error by using [this workaround](https://github.com/dotnet/sdk/issues/1509#issuecomment-323119146) which seems to be a known issue for .net standard and click once.

However the workaround brings in ~100 extra files (mostly DLLs) and about an extra 4MB in size overall.

I'm wondering why the move to net strandard, and also why not package both net standard and .net as before?

Thanks.

**Comment by JoeMayo** [comment]
Closing - Hope the upgrade wasn't too much of a problem for folks.

**Comment by JoeMayo** [comment]
Update on this - I just released LINQ to Twitter v5.1.0. When preparing the release, I updated packaging. Rather than assume everything should work on .NET Standard, the update ensures that .NET Standard, .NET Core, and .NET Framework should be fully supported.
